ui/redux/selectors/content.js

```diff
@@ -123,7 +123,21 @@ export const makeSelectNextUnplayedRecommended = (uri: string) =>
             continue;
           }
 
-          if (!history.some((item) => item.uri === recommendedForUri[i])) {
+          const recommendedUriInfo = parseURI(recommendedUri);
+
+          if (claimsByUri[uri] && claimsByUri[uri].claim_id === recommendedUriInfo.claimId) {
+            // Skip myself (same claim ID)
+            continue;
+          }
+
+          if (
+            !history.some((h) => {
+              const directMatch = h.uri === recommendedForUri[i];
+              const claimNameMatch = h.uri.includes(recommendedUriInfo.claimName);
```

Comment:
I don't like the idea of blindly ignoring claims with identical names. In theory, it should be fine as it's unlikely to occur, but it seems like a potential edge case that could be perceived as a bug at some point down the road in the case a creator  abstractly names all their videos the same thing.
Thinking back on it, I have a seen a few people that post identically posted videos when doing commentary or discussion videos, or sometimes just podcasts without episode numbers. 

just seems like this may lead to issues possibly.

Reply:
`Fixed`

Compare using the somewhat unique short URI instead.